include/xen/be/XenGnttab.hpp

```diff
@@ -92,29 +92,33 @@ class XenGnttabBuffer
 public:
 
 	/**
-	 * @param[in] domId domain id
-	 * @param[in] ref   grant reference id
-	 * @param[in] prot  same flag as in mmap()
+	 * @param[in] domId  domain id
+	 * @param[in] ref    grant reference id
+	 * @param[in] prot   same flag as in mmap()
+	 * @param[in] offset offset to the data inside the buffer
```

Comment:
May be I'm wrong, but I suspect that it's better to use 'offset of' instead of 'offset to'. I'm not sure that 'offset to' is correct construction. Same for line 109.

Reply:
will fix